### DISCUSSION TITLE
Turtles Placed by Turtles Don't Automatically Turn On

### DISCUSSION THREAD

**Discussion by Flexico** [cc-tweaked/CC-Tweaked#944] [Q&A] [answered]
Is this an intended feature? I've made a couple of programs involving a turtle placing other turtles for big projects, but the placed turtles don't actually boot up until I click on them myself. How can I make them automatically start?

**Comment by Lupus590** [comment, marked as answer]
Turtles placed by players don't automatically turn on either, it's when they are first right-clicked that they turn on.

Also, you can wrap computers (which includes turtles) and then use them as a peripheral, one of their peripheral methods allows you to turn them on remotely.

Also, this would be better as a discussion, there should be an option to convert this issue into one somewhere in the right-hand panel thing.

**Reply by Flexico** [reply]
Ah, thank you very much! I love learning new things about this mod~

**Reply by Lupus590** [reply]
Also, a thing for discussions, it's probably a nice thing for future people to have a suggested answer marked as the correct one.
